### ISSUE TITLE
Pokémanki div not centered and too wide when owning many Pokémon

### ISSUE THREAD

**Issue by Exkywor** [zjosua/Pokemanki#11]
#### Describe the bug
When owning many Pokémon, the container in the Stats view becomes too wide and is not center.

#### To Reproduce
1. Own many Pokémon.
2. Open the Stats view.

#### Expected behavior
The Pokémanki title to be fixed, and the rest ot be scrollable horizontally.

#### Screenshots
![image](https://user-images.githubusercontent.com/46603136/194467604-75299e4d-d0d6-4328-b336-2a9d49c8f11a.png)

#### Information about your set-up
*Open Anki, go to Help > About and click on "Copy Debug Info". Paste the result between the backticks below.*
```
    
    Anki 2.1.54 (88e539aa) Python 3.10.7 Qt 6.3.2 PyQt 6.3.1
    Platform: Linux
    Flags: frz=False ao=True sv=3
    Add-ons, last update check: 2022-10-06 14:52:39
    
    
    ===Add-ons (active)===
    (add-on provided name [Add-on folder, installed at, version, is config changed])
    '' ['pokemanki', 0, 'None', '']
    '' ['popup_dictionary', 0, 'None', '']
    Anki Habitica for 21 ['1758045507', 2022-02-09T06:54, 'None', '']
    AnkiDiscord - Discord integration for Anki ['1133851639', 2020-01-13T19:53, 'None', '']
    Button Colours Good Again ['2494384865', 2022-09-27T04:18, 'None', '']
    Enhanced Cloze for Anki 21 ['1990296174', 2022-07-21T03:41, 'None', '']
    More Overview Stats ['2116130837', 2022-09-16T11:56, 'None', '']
    Pull from Duolingo ['557761805', 2022-08-24T21:09, 'None', '']
    Review Heatmap ['1771074083', 2022-06-29T19:43, 'None', '']
    Study Time Stats ['1247171202', 2022-07-26T17:31, 'None', '']
    
    ===IDs of active AnkiWeb add-ons===
    1133851639 1247171202 1758045507 1771074083 1990296174 2116130837 2494384865 557761805
    
    ===Add-ons (inactive)===
    (add-on provided name [Add-on folder, installed at, version, is config changed])
```

What operating system are you using?
- OS: Manjaro, and it also happens on Windows
- version: 2.1.54

#### Additional context
The view was working properly before the following css was introduced to view_trades.cs in commit c83e09d8b6bdf20c771141f264adf073fdaa099a:
```
.pokemanki {
    margin: auto;
    width: fit-content;
}
```
Was there a reason why this was added? Was it not working properly under some setups?

**Comment by zjosua** [comment]
For me it looks like this when I remove the `.pokemanki` CSS rule:
![Screenshot](https://user-images.githubusercontent.com/5960563/194719722-94bbad16-a372-4c92-ad4b-e27efb841582.png)

But as discussed, I'll remove it for now and then we can work from there.

**Comment by EccentricConcentric** [comment]
![2022-11-20 11_42_27-Clipboard](https://user-images.githubusercontent.com/6435037/202920037-3d9b1994-3083-4291-9656-d1a0df1e1c27.png)
![2022-11-20 11_42_59-Statistics](https://user-images.githubusercontent.com/6435037/202920043-b7807d2e-a925-41e8-8260-d63b9d875ec1.png)
I solved this in mine by editing the file in 1041307953\pokemanki_css\view_stats.css such that:

.pk-st-container {
    display: flex;
    /* flex-wrap and justify-content added */
    flex-wrap: wrap;
    justify-content: center;
    gap: 15px;
    padding: 10px;
    margin: 10px 0px;
    overflow-x: auto;
}

Edit: I don't know if this would change anything, but I am using the latest Anki but QT5 (I made some edits to make the addon compatible with QT5).

**Comment by zjosua** [comment]
Some users may wish to have only a single row of Pokémon that is horizontally scrollable.
If someone requests that, we can make the `flex-wrap` configurable.

**Comment by jinny123** [comment]
i'm just a medical student using anki and know nothing about coding, but where in the file would I put that code?

**Comment by LordxRequiem** [comment]
> ![2022-11-20 11_42_27-Clipboard](https://user-images.githubusercontent.com/6435037/202920037-3d9b1994-3083-4291-9656-d1a0df1e1c27.png) ![2022-11-20 11_42_59-Statistics](https://user-images.githubusercontent.com/6435037/202920043-b7807d2e-a925-41e8-8260-d63b9d875ec1.png) I solved this in mine by editing the file in 1041307953\pokemanki_css\view_stats.css such that:
> 
> .pk-st-container { display: flex; /* flex-wrap and justify-content added */ flex-wrap: wrap; justify-content: center; gap: 15px; padding: 10px; margin: 10px 0px; overflow-x: auto; }
> 
> Edit: I don't know if this would change anything, but I am using the latest Anki but QT5 (I made some edits to make the addon compatible with QT5).

I only have a single row of pokemon going vertically. This added line of code didn't help. Is there a way to get it at the end of the stats instead of at the beginning, and also to get the display to wrap appropriately going horizontally? 

![image](https://user-images.githubusercontent.com/120119198/226250942-f14ce858-07da-4be5-8442-1c27c19aa721.png)
